src/site/stages/build/process-cms-exports/tests/entities/paragraph.395b25de-0e29-4e70-929c-629f7c97d33c.json

```diff
@@ -0,0 +1,57 @@
+{
+    "uuid": [
+        {
+            "value": "395b25de-0e29-4e70-929c-629f7c97d33c"
+        }
+    ],
+    "langcode": [
+        {
+            "value": "en"
+        }
+    ],
+    "type": [
+        {
+            "target_id": "link_teaser",
+            "target_type": "paragraphs_type",
+            "target_uuid": "072db7a4-476e-41a6-ab49-c44184281451"
+        }
+    ],
+    "status": [
+        {
+            "value": true
+        }
+    ],
+    "created": [
+        {
+            "value": "2019-03-25T19:27:03+00:00",
+            "format": "Y-m-d\\TH:i:sP"
+        }
+    ],
+    "behavior_settings": [
+        {
+            "value": []
+        }
+    ],
+    "default_langcode": [
+        {
+            "value": true
+        }
+    ],
+    "revision_translation_affected": [
+        {
+            "value": true
+        }
+    ],
+    "field_link": [
+        {
+            "uri": "entity:node\/33",
```

Comment:
Having the `tome-sync` export give us things like this make me a bit uncomfortable - Hopefully a more recent export would give us better data so that we don't have to construct/look up the true urls ourselves.